tests/checkpoint_chunks/run.sh

```diff
@@ -102,7 +102,7 @@ rm -f "$TEST_DIR/cpch.pb"
 
 # Set the failpoint to kill the lightning instance as soon as one chunk is imported
 # If checkpoint does work, this should only kill $CHUNK_COUNT instances of lightnings.
-export GOFAIL_FAILPOINTS='github.com/pingcap/tidb-lightning/lightning/restore/FailIfImportedChunk=return'
+export GOFAIL_FAILPOINTS="github.com/pingcap/tidb-lightning/lightning/restore/FailIfImportedChunk=return($ROW_COUNT)"
```

Comment:
Is it needed to add `row_count` check in `KillIfImportedChunk` failpoint test? https://github.com/pingcap/tidb-lightning/blob/7bae12ebe2b6ca1c3c712eb35b7f0c39484c86a5/lightning/restore/restore.go#L390-L396

Reply:
It's not a cause of failure but I'm applying this anyway.